### DISCUSSION TITLE
SQL工单个人企微通知如何设置？

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2524] [Q&A]
1、Archery工单状态变化时，可以发送企业微信通知给SQL工单的提交人吗？是在用户管理里面添加用户的 UserID ? 
2、求教下这个UserID 在哪获取的，谢谢。
![image](https://github.com/hhyo/Archery/assets/32159630/25a258b4-0ae4-4baa-9b72-fd88bad14106)

**Comment by LeoQuote** [comment]
搜一下不难吧，咋不搜一下呢？ https://open.work.weixin.qq.com/help/wap/detail?docid=14973&helpType=undefined&replykey=undefined&person_id=1

**Reply by a8159236** [reply]
您好，感谢回复，这边有看到这个网页，但是找管理员要了账号(userid)，但是配置进去后，sql提交人的工单状态从待审核变成审核通过后，依然没有收到企微通知是什么原因呢？![image](https://github.com/hhyo/Archery/assets/32159630/ebe7873d-5dfa-42a1-ae2e-2e253e987527)

**Reply by LeoQuote** [reply]
首先是可以用master分支的镜像或者代码重试一下，然后是请看一下日志

**Reply by a8159236** [reply]
好的谢谢
